packages/cos/build.yaml

```diff
@@ -22,6 +22,9 @@ requires:
 - name: "selinux-policies"
   category: "system"
   version: ">=0"
+- name: "yq"
```

Comment:
This should go into the `copy` part, otherwise it will bring `golang` inside `system/cos` (the installable artifact)

Reply:
good point! probably an artifact of my rebase!

Reply:
ah no, this was added by myself